### PR TITLE
Add Auras for Inspiring/Dread Marshal Stances

### PR DESCRIPTION
**Pull request by SpartanCPA**

### CODE REVIEW

File: packs/data/feats.db/marshal-dedication.json
Comment (SpartanCPA) [inline packs/data/feats.db/marshal-dedication.json:78]: @stwlam I'd like your input here. Adding one of the new effects above properly turns off the 10 foot aura RE above this one, but they don't turn this aura RE on.

If a Critical Success occurs using either stance, the whole aura should increase to 20 feet, so I'm trying to have this new RE turn on and set itself to the size of the stance, but it won't do it. Probably a timing issue?

File: packs/data/feats.db/marshal-dedication.json
Comment (SpartanCPA) [inline packs/data/feats.db/marshal-dedication.json:71]:
```suggestion
                "radius": "{item|flags.pf2e.rulesSelections.marshalAuraSize}",
```

**Review by TikaelSol — APPROVED** [review]
Good to go pending the comment for shark.

**Comment by stwlam** [comment]
Can we hold off on this? A better implementation will be available soon.

**Comment by TikaelSol** [comment]
Closing for better implementation later

**Pull request closed without merging**
